### DISCUSSION TITLE
I can't find a follow-up to the support statuses of each version of BS.

### DISCUSSION THREAD

**Discussion by wnunezc** [twbs/bootstrap#39644] [Q&A]
I can't find a follow-up to the support statuses of each version of BS.

I understand that the latest version of BS4: 4.6.2 but it is not clear to me what status the support is in. 

if you are still receiving some type of care from the community.

I ask this because recently with the latest Google actions the following warning is appearing:

> CSS appearance values inner-spin-button, media-slider, media-sliderthumb, media-volume-slider, media-volume-sliderthumb, push-button, searchfield-cancel-button, slider-horizontal, sliderthumb-horizontal, sliderthumb-vertical, square-button are not standardized and will be removed.

When checking that CSS is related I see the following:

```
[type="number"]::-webkit-inner-spin-button,
[type="number"]::-webkit-outer-spin-button {
  height: auto;
 }
```

It is not clear to me if a version 4.6.3 will come to correct the loss of compatibility of some existing CSS in this version of BS vs the list of deprecations that Google Chrome introduces.

Taking into account the volume of projects that use BS4, for some it is not yet feasible to switch to the BS5 version, I have not even verified if it also has that warning.

Link reference: https://chromestatus.com/feature/5066630972833792

**Comment by julien-deramond** [comment]
Hi @arcanisgk 
Bootstrap v4 reached its end-of-life (EOL) on January 1st, 2023 (more info at https://github.com/twbs/release - that we need to update to include info about v5 as well). With this EOL status, there will be no further releases for Bootstrap v4.

**Reply by wnunezc** [reply]
then all versions prior to 5.x ... will be unusable.
